lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -662,6 +662,24 @@ func (c *Client) handleRDPFastPathPDU(data []byte) C.CGOErrCode {
 	return C.ErrCodeSuccess
 }
 
+//export handle_rdp_channel_ids
+func handle_rdp_channel_ids(handle C.uintptr_t, io_channel_id C.uint16_t, user_channel_id C.uint16_t) C.CGOErrCode {
+	return cgo.Handle(handle).Value().(*Client).handleRDPChannelIDs(io_channel_id, user_channel_id)
+}
+
+func (c *Client) handleRDPChannelIDs(io_channel_id, user_channel_id C.uint16_t) C.CGOErrCode {
```

Comment:
```suggestion
func (c *Client) handleRDPChannelIDs(ioChannelID, userChannelID C.uint16_t) C.CGOErrCode {
```

For Go functions, let's follow Go naming conventions.

It's fine if the `//export`ed C func uses different conventions.